docker/wfm/app/entry-point.sh

```diff
@@ -19,5 +19,21 @@ set -eu ${DEBUG:+-x}
 PATH=${PATH}:/opt/storm/bin
 
 cd /app
-make kill-all
-exec make deploy-all
+
+case ${WFM_TOPOLOGIES_MODE:-} in
+
+  blue)
+    make kill-all-blue
+    exec make deploy-all-blue
+    ;;
+
+  green)
+    make kill-all-green
+    exec make deploy-all-green
+    ;;
+
+  *)
+    make kill-all
```

Comment:
Unexpected behavior, at least for me. You completely ignore the value of WFM_TOPOLOGIES_MODE. Such behavior can mask incorrect values into WFM_TOPOLOGIES_MODE.

Reply:
I left it for backward compatibility. If we do not pass WFM_TOPOLOGIES_MODE typologies will run on blue mode